Description: New kernel 5.2 moved around some definitions of constants, adding the necessary includes
Author: Gianfranco Costamagna <locutusofborg@debian.org>

---
Forwarded: no
Last-Update: 2019-07-19

--- linux-atm-2.5.1.orig/src/arpd/arp.c
+++ linux-atm-2.5.1/src/arpd/arp.c
@@ -17,6 +17,7 @@
 #include <netinet/in.h> /* for ntohs, etc. */
 #define _LINUX_NETDEVICE_H /* very crude hack for glibc2 */
 #include <linux/types.h>
+#include <linux/if.h>
 #include <linux/if_arp.h>
 #include <linux/if_ether.h>
 #include <atm.h>
--- linux-atm-2.5.1.orig/src/arpd/itf.c
+++ linux-atm-2.5.1/src/arpd/itf.c
@@ -14,6 +14,7 @@
 #include <sys/socket.h>
 #define _LINUX_NETDEVICE_H /* glibc2 */
 #include <linux/types.h>
+#include <linux/if.h>
 #include <linux/if_arp.h>
 
 #include "atmd.h"
--- linux-atm-2.5.1.orig/src/maint/atmdump.c
+++ linux-atm-2.5.1/src/maint/atmdump.c
@@ -14,6 +14,7 @@
 #include <sys/types.h>
 #include <sys/time.h>
 #include <sys/socket.h>
+#include <linux/sockios.h>
 #include <netinet/in.h> /* for htonl and ntohl */
 #include <atm.h>
 
--- linux-atm-2.5.1.orig/src/maint/saaldump.c
+++ linux-atm-2.5.1/src/maint/saaldump.c
@@ -15,6 +15,7 @@
 #include <sys/time.h>
 #include <sys/types.h>
 #include <sys/socket.h>
+#include <linux/sockios.h>
 #include <atm.h>
 
 #include "pdu.h"
